woocommerce/woocommerce-ios
# Analytics Hub: Add Main View Model

part of #8148 

# Why

This PR adds an `AnalyticsHubViewModel` filled with dummy data in order to add a base so we can begin expanding it further.


# How

- Adds an `AnalyticsReportCardViewModel` type so we can transmit data easily to `AnalyticsReportCardView`
- Adds `AnalyticsHubViewModel` with revenue and orders dummy data.
- Upates `AnalyticsHubView` to read data from its view model. 


# Screenshot

<img width="427" alt="screenshot" src="https://user-images.githubusercontent.com/562080/203439730-cb854436-8a24-4645-a22e-8d0ec2f95b48.png">

# Testing 

- Launch the app
- Tap the "See More" button from the main dashboard
- See the report cards with dummy data.

---
